target/linux/ramips/image/rt288x.mk

```diff
@@ -30,6 +30,22 @@ define Device/ar725w
 endef
 TARGET_DEVICES += ar725w
 
+define Device/dap-1522-a1
+  DTS := DAP-1522-A1
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := $(ralink_default_fw_size_4M)
```

Comment:
ralink_default_fw_size_4M (3866624) doesn'T match the size if your factory partition (0x3a0000 == 3801088 bytes)